packages/calypso-e2e/src/lib/components/editor-gutenberg-component.ts

```diff
@@ -95,13 +94,14 @@ export class EditorGutenbergComponent {
 		if ( await emptyParagraphLocator.count() ) {
 			await emptyParagraphLocator.click();
 		} else {
-			emptyBlockLocator.click();
+			await emptyBlockLocator.click();
 		}
 
-		for await ( const line of splitText ) {
+		for ( const line of lines ) {
 			// Select the last Paragraph block which is empty.
 			const locator = this.editor.locator( selectors.paragraphBlock( { empty: true } ) ).last();
-			await locator.fill( line );
+
+			await locator.type( line );
```

Comment:
**question (non-blocking):** I see that we often use `locator.fill`. Do we need to change it to `locator.type`?

Reply:
Excellent question Karen! I'm not @WunderBart but I can help answer this. 

Type would indeed better simulate user input. IIRC it also fires the keyboardevent as well. With that said, Type can take much longer than Fill since it actually types out exchange character. Setting the delay to a smaller value would help somewhat, but Fill still wins hands down in terms of speed. 

When the framework was initially constructed the decision taken was to use Fill instead of Type due to reasons cited above. It doesn't mean we are not open to revisiting these decisions though!

Reply:
Ah, good catch! I changed it for more granular control of the user input – needed to verify if the flakiness didn't come from the editor not catching up with the `fill` speed. I decided to leave it despite the fact `fill` turned out not to be the culprit, as it felt closer to how a user would interact with the editor. I didn't consider the speed factor, though, and now I can see that, e.g., the Web App E2E suite runs significantly longer (~1:30@trunk vs. ~2:00@fix/e2e-atomic/editor-revisions). Because of that and since `fill` isn't causing any issues, I'd revert it for speed's sake. Thanks, @worldomonation for pointing it out! 🙇

Reply:
Reverted in https://github.com/Automattic/wp-calypso/pull/69345/commits/4cb83720b4edcaf2627ef228f109853ac3bd45f3